README.md

```diff
@@ -108,8 +129,29 @@ Destroy the local workspace and it's corresponding state file on mgmt
 make terraform--destroy TERRAFORM_WORKSPACE="<YOUR_SHORTCODE_AND_JIRA_NUMBER>" # Will attempt to login to AWS first using SSO
 ```
 
+### Automated workspace destroys
+
+We have some automated behaviour for destroying our workspaces - we build off a new commit each time we deploy and there are times where we forget to destroy old workspaces or have CI workspaces be removed. So we have two processes:
+
+#### Destroy Expired Workspaces
+
+- This is run on a cron job as a github workflow every evening and scans the workspaces in dev (local deployments) and ref (CI deployments)
+- Each deployment is deployed with a resource group that has an expiration date on it, the workspace is destroyed if its past that date
+- Each deploy or change within a workspace extends the expiration date
+- Can be run adhoc by using the command: `make destroy--expired TERRAFORM_WORKSPACE="dev"`
+
+#### Destroy Redundant Workspaces
+
+- This is part of the Pull Request github workflow and checks to destroy previous commits on the same workspace leaving only the latest
+- When you run locally you can override this with the `KILL_ALL` flag which destroys the current commit too
+- Can be run adhoc with the command: `make destroy--redundant-workspaces BRANCH_NAME=mybranch DESTROY_ALL_COMMITS_ON_BRANCH=false KILL_ALL=false`
+
 ## Updating Roles
 
+We have several roles that we currently handle outside of terraform as its needed to deploy the terraform:
+
+-
```

Comment:
Should this state the roles we have outside of terraform?

Reply:
Yes it should 😂